### DISCUSSION TITLE
apps.plugin can exclude parents and childs process?

### DISCUSSION THREAD

**Discussion by yezhengmao1** [netdata/netdata#13385] [Q&A] [answered]
I found the apps.plugin include all parents and childs process, it`s hard to analyze the process`s metrics.
Any method to exclude it?

**Comment by ilyam8** [comment, marked as answer]
Hi, @yezhem. See https://github.com/netdata/netdata/tree/master/collectors/apps.plugin#configuration

> For each process given, its whole process tree will be grouped, not just the process matched. The plugin will include both parents and children. If including the parents in the group is undesirable, the line other: * should be appended to the apps_groups.conf.

**Reply by yezhengmao1** [reply]
Thanks, Its meaning I should put parents process name in other:*?

**Reply by ilyam8** [reply]
No, you shouldn't. Just `other: *`.
